lib/fake_stripe/stub_stripe_js.rb

```diff
@@ -22,6 +22,14 @@ class StubStripeJS < Sinatra::Base
       IO.read(file_path) + IO.read(mock_file_path)
     end
 
+    get '/v3/' do
```

Comment:
Just for consistency within this diff -- you corrected `'` to `"` in another file, might want to stick with `"` in additions to this file.